src/evm_transition_tool/transition_tool.py

```diff
@@ -286,23 +324,30 @@ def evaluate(
         )
 
         if debug_output_path:
+            t8n_call = " ".join(args)
+            t8n_output_base_dir = os.path.join(debug_output_path, "t8n.sh.out")
+            if self.trace:
+                t8n_call = t8n_call.replace(temp_dir.name, t8n_output_base_dir)
             t8n_script = textwrap.dedent(
                 f"""\
                 #!/bin/bash
-                mkdir {temp_dir.name}
-                {' '.join(args)} < {debug_output_path}/stdin
+                rm -rf {t8n_output_base_dir}
```

Comment:
I feel kinda uneasy of adding this line to the script, it could be really bad if a typo would result in `t8n_output_base_dir=='/'`.

Reply:
Thanks for spotting this! See 3070d94b. This hardcodes the "t8n.sh.out" directory (instead of f-string variable substitution), so that we are only at risk of deleting these directories, no matter where they're located (or what the value of `debug_output_path` is).

Reply:
Update: See 3070d94b and comment above on same issue with transition_tool.py.